### ISSUE TITLE
QR generation should be asynchronous

### ISSUE THREAD

**Issue by dnna** [dnna/pixelbonus#7]
We can implement using a queuing system like [BCCResqueBundle](https://github.com/michelsalib/BCCResqueBundle)

**Comment by dnna** [comment]
Turns out this is no longer needed after #8. Codes are generated instantaneously now (I tested generating 200 codes and it took <3 seconds) so it does not need asynchronous processing.
